bootstrap-joining-demo/marathon-api/marathon/app.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
@longshorej Just a note. This example demonstrates HOST mode. We could add bridge-mode example later, when my PR is merged: https://github.com/akka/akka-management/pull/99

Reply:
Could you PR that and update the readme here or else where discussing when what mode should be used? I'm not sure what those modes imply